src/agent/protocols/build.rs

```diff
@@ -107,7 +107,7 @@ fn real_main() -> Result<(), std::io::Error> {
         .for_each(|p| println!("cargo:rerun-if-changed={}", &p));
 
     let ttrpc_options = Customize {
-        async_server: true,
+        async_all: true,
```

Comment:
There are still some concerns about adding async support to the runtime. Please either make `async_client` optional or get a consensus about async runtime before making such a change.

Reply:
Add #[cfg(feature = "async")] to control whether to use sync or async